src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1764,7 +1770,7 @@ else if (clickedBlock != null &&
         }
 
         //otherwise apply rules for buttons and switches
-        else if (clickedBlock != null && instance.config_claims_preventButtonsSwitches && (clickedBlockType == null || clickedBlockType == Material.STONE_BUTTON || Tag.BUTTONS.isTagged(clickedBlockType) || clickedBlockType == Material.LEVER))
+        else if (clickedBlock != null && instance.config_claims_preventButtonsSwitches && (clickedBlockType == null || clickedBlockType == Material.STONE_BUTTON || clickedBlockType == Material.WARPED_BUTTON || clickedBlockType == Material.CRIMSON_BUTTON || Tag.BUTTONS.isTagged(clickedBlockType) || clickedBlockType == Material.LEVER))
```

Comment:
tags/blocks/buttons includes stone_button, polished_blackstone_button (which this missed) and #wooden_buttons, which includes all wood buttons, crimson, and warped. Should actually just remove check for Material.STONE_BUTTON instead of adding additional materials.

Reply:
> tags/blocks/buttons includes stone_button, polished_blackstone_button (which this missed) and #wooden_buttons, which includes all wood buttons, crimson, and warped. Should actually just remove check for Material.STONE_BUTTON instead of adding additional materials.

idk... it is your decision, I did what I told